MachObfuscator/Options/Options.swift

```diff
@@ -1,16 +1,29 @@
 import Foundation
 
+class EraseSectionConfiguration {
+    let sectionName: String
+    let segmentName: String
+
+    init(_ sectionDef: String) {
+        let sectionParts = sectionDef.split(separator: ",", maxSplits: 2)
```

Comment:
What do you think about adding some user friendly check:?
```
guard sectionParts.count == 2 else {
    fatalError("Section must by pointed with SEGMENT,SECTION format")
}
```

This kind of check could feel inelegant, but we could hide it a bit by making this initialiser fileprivate (or making extension private).

Reply:
Check added. I used private extension.